modules/overrides/runtime_config_overrides.go

```diff
@@ -277,7 +278,7 @@ func (o *runtimeConfigOverridesManager) GetTenantIDs() []string {
 		return nil
 	}
 
-	return maps.Keys(tenantOverrides.TenantLimits)
+	return slices.Collect(maps.Keys(tenantOverrides.TenantLimits))
```

Comment:
it seems like `slices.Collect` calls result in repeated appends to to an empty slice which is a bit unfortunate.

https://cs.opensource.google/go/go/+/refs/tags/go1.23.2:src/slices/iter.go;l=49-54

wdyt about pre-allocing the slice to the appropriate capacity from the map and then using AppendSeq?

Reply:
not sure if this is going to be measurable, but why not. At least that should be closer to https://cs.opensource.google/go/x/exp/+/701f63a6:maps/maps.go;l=21